index.js

```diff
@@ -30,11 +31,9 @@ function NYC (opts) {
   config = config.nyc || {}
 
   // load exclude stanza from config.
-  this.exclude = config.exclude || ['node_modules[\/\\\\]', 'test[\/\\\\]', 'test\\.js']
+  this.exclude = ['node_modules/**'].concat(config.exclude || ['test/**', 'test.js'])
```

Comment:
Agreed, it seems reasonable to assume we'd never want to instrument in `node_modules` regardless of the nesting.